pkg/phlaredb/block_querier.go

```diff
@@ -405,6 +423,19 @@ func InRange(q Querier, start, end model.Time) bool {
 	return true
 }
 
+type ReadAPI interface {
```

Comment:
This type is here strictly to enforce `Queriers` implements these methods, correct? I don't see it used anywhere else.

Reply:
Yeah we don't really need it, but I think this is the start of a bigger refactoring, we in fact should only implement this in Queriers.